KFMOTD.ini

```diff
@@ -1,7 +1,9 @@
 [KFGui.KFMOTD]
 # This is where the HTML News page is hosted at. To create your own page, you need to go here
 # https://pastebin.com/YZar5TAb And follow the exact format. No JS Scripts or others, just use HTML structure like that
-# Then page ID (from your address bar) in place of only 'YZar5TAb'
-getRequest="GET /raw/YZar5TAb HTTP/1.1"
+# Then page ID (from your address bar) in place of only '/raw/wYZar5TAb'
+
+# Pastebin returns shit if we using old 1.1 proto, I used 2.0 and everything is ok now
+getRequest="GET /raw/YZar5TAb HTTP/2.0"
```

Comment:
You mentioned that it works now -- Can you confirm it still has the same layout? A screenshot maybe 😃 Then we can merge and change the Steam comments/description.